packages/electrode-archetype-opt-less/README.md

```diff
@@ -0,0 +1,41 @@
+# Electrode Archetype Option Stylus
```

Comment:
Entire document refers to stylus instead of less, including the optional parameter name and package name